frontend/src/components/layouts/Footer/index.js

```diff
@@ -59,18 +60,21 @@ const Footer = () => {
                     <Divider size={1} />
                     <ExternalLink
                         theme="dark"
-                        href="https://hackjunction.com/terms"
+                        href={config.PLATFORM_OWNER_WEBSITE + '/terms'}
```

Comment:
I would rather have this as `config.PLATFORM_OWNER_TERMS_URL`, to not force the url to end with /terms